x-pack/plugins/monitoring/server/lib/logstash/get_pipeline.js

```diff
@@ -17,13 +17,18 @@ export function _vertexStats(vertex, vertexStatsBucket, totalProcessorsDurationI
   const isInput = vertex.plugin_type === 'input';
   const isProcessor = vertex.plugin_type === 'filter' || vertex.plugin_type === 'output';
 
+  const timeseriesIntervalInMillis = timeseriesIntervalInSeconds * 1000;
+
   const eventsInTotal = vertexStatsBucket.events_in_total.value;
   const eventsOutTotal = get(vertexStatsBucket, 'events_out_total.value', null);
 
   const durationInMillis = vertexStatsBucket.duration_in_millis_total.value;
 
   const inputStats = {};
   const processorStats = {};
+  const eventsProcessedStats = {
+    events_out_per_millisecond: eventsOutTotal / timeseriesIntervalInMillis
```

Comment:
Is there an `eventsOutTotal` for output vertices?

Reply:
I believe that we are passing it, as evidenced here:

<img width="650" alt="screen shot 2018-05-25 at 1 42 13 pm" src="https://user-images.githubusercontent.com/18429259/40558501-7fb64fb4-6021-11e8-8ee7-2e039ba0df66.png">

`eventsOutTotal` is passed to the client as `events_out`

Reply:
Hmmm, yeah, good point. All good here!